src/controls/tab-bar.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import { fieldOptionsType } from '../forms/helpers/field-prop-types'
+import { objectify, noop } from '../utils'
+
+/**
+ *
+ * @name TabBar
+ * @type Function
+ * @description A control component for navigating among multiple tabs
+ * @param {Boolean} [vertical] A boolean to determine whether the tabs should be aligned vertically or horizontally (optional, default `false`)
+ * @param {Array} [options] An array of tab values (strings or key-value pairs)
+ * @param {String|Number} [value] - The value of the current tab
+ * @param {Function} [onChange] - A function called with the new value when a tab is clicked
+ * @example
+ *
+ * function ShowTabs ({ tabs, currentTab, changeCurrentTab }) {
+ *   return (
+ *     <div>
+ *       <TabBar
+ *         vertical={false}
+ *         options={tabs}
```

Comment:
I'd make this a constant rather than a prop to indicate what options typically look like.